common/prettify.pm

```diff
@@ -10,6 +10,7 @@ use warnings;
 
 use FileHandle;
 use Cwd;
+our $path = "";
```

Comment:
Where is this used?

Reply:
In junit log_file property.

Reply:
The variable is assigned a value on line 25.  I can't see any place where it's actually used.

Reply:
used on Line 644, in Prettify::JUnit::Footer ()

Reply:
Line 644:    print $out $indent x 3, "<property name=\"log_file\" value=\"$Prettify::Full_HTML::path\"/>\n";